### ISSUE TITLE
question about custom object

### ISSUE THREAD

**Issue by showkeyjar** [dmlc/xgboost#7315]
I want custom a xgboost object function, but I don't want custom second derivative.

how can I do that use python?

for example, a regression problem:

`
  def get_loss(distance):
      distance = np.where(distance==0, 0.5, distance)
  
      def date_normal_loss(preds, dtrain):
          labels = dtrain.get_label()
          loss = (np.square(np.subtract(labels,preds)).mean() / np.abs(distance)) * max(distance)
          return loss
      return date_normal_loss, xgboost_hessian_result(preds, dtrain)
`
how to call xgboost default hessian functions?
